docs/UserGuide.md

```diff
@@ -237,6 +248,17 @@ The subsequent pairs should not have a `+` sign before them. i.e. `r/+CS1101S +M
 - You only need to specify one `r/`. i.e. `r/+CS1101S r/+MA1521-TA` is unnecessary and will cause an error.
 </box>
 
+<box type="warning" seamless>
+
+  **Reminder**: the other meaning of ``+``
+
+* Items in **square** brackets and with `+` after them can be used zero or more times.<br>
+  e.g. `[t/TAG]+` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+
+* Items in **round** brackets and with `+` after them can be used one or more times.<br>
+  e.g. `(t/TAG)+` can be used as `t/friend`, `t/friend t/family` etc.
+</box>
+
```

Comment:
I think just put it right below the format and say
Please note that the first `+` should be typed as is while the second `+` is the multiplicity syntax explained [here](link)